src/pages/PlanPage/subRoutes/CurrentOrgPlan/LatestInvoiceCard/LatestInvoiceCard.spec.tsx

```diff
@@ -16,7 +16,9 @@ const invoice = {
 }
 
 describe('LatestInvoiceCard', () => {
-  function setup({ invoices }) {
+  function setup({ invoices }: { invoices: any }) {
```

Comment:
I think you can do 
`const mockedInvoices = useInvoices as jest.Mock 
`
and 
`mockedInvoices.mockReturnValue(...)
`

Look up `as jest.Mock` for examples in code

Reply:
will try and report back! Good looks

Reply:
worked! fixed in [b28b6e3](https://github.com/codecov/gazebo/pull/2642/commits/b28b6e31f81f61cabe9e877e11fbbf703e650002)